modules/gltf/src/lib/extensions/deprecated/EXT_feature_metadata.ts

```diff
@@ -170,6 +219,41 @@ function getStringOffsets(
   return null;
 }
 
+/**
+ * Parse numeric property values
+ * @param valuesDataBytes - values data array
+ * @param propertyType - type of the property
+ * @param elementCount - number of rows in the featureTable
+ * @returns Number data in a typed array
+ */
+function getNumericAttributes(
+  valuesDataBytes: Uint8Array,
+  propertyType:
+    | 'INT8'
```

Comment:
This is getting very verbose, you should probably define some types.
```typescript
export type I3SPropertyType = 
   | 'INT8'
    | 'UINT8'
    | 'INT16'
    | 'UINT16'
    | 'INT32'
    | 'UINT32'
    | 'INT64'
    | 'UINT64'
    | 'FLOAT32'
    | 'FLOAT64'
    | 'BOOLEAN'
    | 'STRING'
    | 'ENUM'
    | 'ARRAY';

export type I3SPropertyIntegerType = 
   | 'INT8'
    | 'UINT8'
    | 'INT16'
    | 'UINT16'
    | 'INT32'
    | 'UINT32'
    | 'INT64'
    | 'UINT64'
    | 'FLOAT32'
    | 'FLOAT64;
```

Reply:
Done